console2/src/api/process/index.ts

```diff
@@ -265,10 +265,15 @@ export const get = (
     return fetchJson(`/api/v2/process/${instanceId}?${params.toString()}`);
 };
 
-export const getRoot = (
-    instanceId: ConcordId
-): Promise<ProcessEntry> => {
-    return fetchJson(`/api/v1/process/${instanceId}/root`);
+export const getRoot = async (instanceId: ConcordId): Promise<ProcessEntry|void> => {
+    const resp = await managedFetch(`/api/v1/process/${instanceId}/root`, { method: 'GET' });
+    if (resp.status === 200) {
+        console.log(resp);
```

Comment:
Debug logging? You should see the response in the dev tools anyway.

Reply:
indeed, wasn't intended to stick around